agr_java_core/src/main/java/org/alliancegenome/api/entity/DiseaseAnnotationDocument.java

```diff
@@ -22,13 +22,13 @@
 public class DiseaseAnnotationDocument extends SearchableItemDocument {
 
 	private String uniqueId;
-	private VocabularyTerm diseaseRelation;
+	private VocabularyTerm relation;
 	private DOTerm object;
 	private List<ECOTerm> evidenceCodes;
 	private Reference singleReference;
 	private Set<Reference> references;
 
-	private String diseaseRelationNegation;
+	private String relationNegation;
```

Comment:
I realize that the way this was initially implemented was with certain names. I would prefer not adding "negation" to everything. Just call it what it is: "relation". And if we need a "generated" string then call it that "relationText" or "relationGeneratedText" etc. And if we need to "negate" the text then call a method that has an appropriate name "negateRelationText".
We use the wrapper class "DiseaseAnnotationDocument" for convenience. That way the documents going into ES are much smaller than the object coming out of the curation system. We use only the fields needed to show the table and nothing more. To me this change is fine the way it is, however I would like to see better naming but I will leave that up to @markquintontulloch

Reply:
Yeah, I'm not a fan of diseaseRelationNegation / relationNegation either - I think this implies it is always the negated version whereas it's actually the generated string for both positive and negative relations.
If we're going to cause breaking changes in the UI then I'd prefer renaming to something more descriptive as @oblodgett describes above.  However, I'd prefer not to rename it to "relationNegation" as currently proposed since it is no more descriptive, has nothing to do with the LinkML, and will require UI changes.

Reply:
The UI changes for this shouldn't be that extensive, as can be seen here: https://github.com/search?q=repo%3Aalliance-genome%2Fagr_ui%20diseaseRelation&type=code and I would also agree that it should reflect LinkML whenever possible.

Reply:
I hadn't spotted the direct reference to `diseaseRelation` in the UI code in AnnotatedEntitiesPopupCuration.js.  Given that, my vote would definitely be to replace `diseaseRelationNegation` with something more descriptive than `relationNegation` and then replace `diseaseRelation` with `relation` everywhere else as in the current PR.